build_debian.sh

```diff
@@ -271,6 +271,8 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install docker-ce=${DOCKER_VERSIO
 
 install_kubernetes () {
     local ver="$1"
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-get update
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install gnupg
```

Comment:
>gnupg [](http://example.com/codeflow?start=59&length=5)

This will install more package(s) into sonic image. Is it really needed? Your intention is to change master image only.

Reply:
> > gnupg
> 
> This will install more package(s) into sonic image. Is it really needed? Your intention is to change master image only.

Yes, indeed need to install gnupg before install gpg file for bookworm, I have removed gnupg after installing the gpg file.